css/stylesheet.css

```
@@ -31,37 +36,115 @@ header {
     left: 50%;
     transform: translate(-50%);
     width: 460px;
-    height: 143px;
 }
-    
+
 .worker-container {
     background-color: #e6e6fa;
     width: 50%;
-    height: 900px;
+    padding: 100px 40px 80px 40px;
     text-align: center;
 }
     
 .client-container {
     background-color: #d8bfd8;
     width: 50%;
-    height: 900px;
+    padding: 100px 40px 80px 40px;
     text-align: center;
 }
 
 .mobile {
-    height: 400px;
-    margin-top: 100px;
-    position:relative;
+    max-width: 386px;
+    display: block;
+    margin: 0 auto;
```

.mobile のPC時はwidth固定（参考サイトだとそうだと思います）
max-widthにしてしまうと、ブラウザ幅によって縮小してしまいます。

```suggestion
    width: 386px; //widthは固定
    display: block;
    margin: 0 auto;
```